### ISSUE TITLE
Bug in detection urls

### ISSUE THREAD

**Issue by Urkman** [fweinaug/expandable_text#18]
There is currently a bug in detection urls.
A text like "test..." is detected as a url.

**Comment by Urkman** [comment]
Screenshot...
<img width="411" alt="Bildschirmfoto 2021-05-27 um 11 09 14" src="https://user-images.githubusercontent.com/94297/119799063-06236e80-bedc-11eb-8b3b-c3d1b8cce7de.png">

**Comment by Urkman** [comment]
Perhaps use this RegEx: 
```
r'(?:(?:https?|ftp):\/\/)?[-a-z0-9@:%._\+~#=]{1,256}\.[a-z0-9]{1,6}(\/[-a-zA-Z0-9()@:%_\+.~#?&\/=]*)?'
```

**Comment by MyisCARRY** [comment]
I wrote a possible solution to this issue in #21

**Comment by fweinaug** [comment]
thank you for creating the issue! I just released version 2.3 which should fix this problem by using your regex.
